exporter/views.py

```diff
@@ -35,13 +37,81 @@ def get_original_asset_id(download_url):
     """
     if download_url.startswith("http://tile.loc.gov/"):
         pattern = r"/service:([A-Za-z0-9:\-]*)/"
-        asset_id = re.search(pattern, download_url)
+        asset_id = re.search(pattern, download_url).group(1)
         assert asset_id
-        return asset_id.group(1)
+        logger.debug("Found asset ID %s in download URL %s" % (asset_id, download_url))
```

Comment:
Since this will be called potentially very many times, I would not force string interpolation:

```suggestion
        logger.debug("Found asset ID %s in download URL %s", asset_id, download_url)
```